doc: fix a wrong description for report -s

qpakzk opened the pull request:
The commit 2bd1bbe (tests: Add test cases for record with -f option.) updated a wrong description for -s option in report.

Signed-off-by: Sangwon Hong <qpakzk@gmail.com>

honggyukim commented on doc/uftrace-report.md:
```diff
@@ -32,11 +32,12 @@ REPORT OPTIONS
-    Note that the first 3 keys should be used when
-    neither of `--avg-total` nor `--avg-self` is used.  Likewise, the last 3
-    keys should be used when either of those options is used.
+    `total-max`, `self` (time), `self-avg`, `self-min`, `self-max`, `call`,
+    `func`, `avg`, `min` and `max`.
+    Note that the first 9 keys (total~call) should be used when
+    neither of `--avg-total` nor `--avg-self` is used.
+    Likewise, the last 3 keys (avg~max) should be used
+    when either of those options is used.
```
How about this description?
```
-s *KEYS*[,*KEYS*,...], \--sort=*KEYS*[,*KEYS*,...]
:   Sort functions by given KEYS.  Multiple KEYS can be given, separated by
    comma (,).  Possible keys are `total` (time), `total-avg`, `total-min`,
    `total-max`, `self` (time), `self-avg`, `self-min`, `self-max`, `call`
    and `func`.  But if either `--avg-total` or `--avg-self` is used, the
    possible keys can be `avg`, `min` and `max` that apply to total or self
    time respectively.
```

honggyukim reviewed (commented):
Thanks for the fix! Could you update the same change to `doc/ko/uftrace-report.md` as well?

qpakzk commented:
@honggyukim Thanks for your comment.
I updated the commit to reflect your advice.
I also modified the description for the Korean version.
Furthermore, I corrected a minor grammar error.

honggyukim reviewed (approved):
Thanks very much for the update. I'm okay with this.

namhyung reviewed (approved):
LGTM
